portal-backend/loader/dataset_loader/single_input_file_dependency_loader.py

```diff
@@ -50,9 +50,6 @@ def compound_dose_lookup(x):
     if DependencyEnum.is_compound_experiment_enum(dataset_enum):
         entity_lookup = compound_experiment_lookup
         entity_type = "compound_experiment"
-    elif DependencyDataset.is_dose_enum(dataset_enum):
-        entity_lookup = compound_dose_lookup
-        entity_type = "compound_dose"
```

Comment:
This makes me wonder, after deleting the repurposing secondary dose dataset, is "compound_dose" still a used/valid entity_type? And if it's not a valid entity_type, can the CompoundDose sqlite table be deleted, too?

Reply:
Oh good point, thank you! Yeah it doesn't look like it's being used at all anymore.